index.html

```diff
@@ -1421,7 +1417,21 @@ <h2>Accessibility</h2>
 		<section id="ebrl-packaging">
 			<h2>Packaging</h2>
 
-			<p>A packaged eBraille file set MUST use the extension <code>.ebrl</code>.</p>
+			<p>When [=eBraille publications=] are distributed to end users as a single file set, they MUST be packaged
+				in an EPUB container, as defined in <a data-cite="epub-33#sec-container-zip">OCF ZIP Container</a>
+				[[epub-33]].</p>
+
+			<p>Packaged eBraille publications are also identified by the media type identifier
+					"<code>application/epub+zip</code>" [[epub-33]]. The requirements for specifying the identifier in
+				the <code>mimetype</code> file are defined in <a data-cite="epub-33#sec-zip-container-mime">OCF ZIP
+					container media type identification</a>.</p>
+
+			<p>Unlike EPUB, which does not require a file extension, a packaged eBraille file set MUST use the extension
```

Comment:
I didn't know that EPUB's didn't required .epub.  Should we at least say although the recognized extension is .epub?
just seems like an obvious question folks will have when reading this statement.

Reply:
I don't think we need to bring in epub here. The requirement is a .ebrl extension, full stop. I've added a note to explain that if the file has a .epub extension it won't be an ebraille file anymore but might open in an epub reading system with some loss of functionality. That might be more useful to help people understand how the two relate.